### PR TITLE
Add support for old job queue

### PR DESCRIPTION
**Pull request by jprochazk**
https://keboola.atlassian.net/browse/PSGO-58

### CODE REVIEW

**Comment by jprochazk** [comment]
@michaljurecko nevíš proč padá ten clean project?

**Comment by michaljurecko** [comment]
:eyes:

File: pkg/keboola/queue_job.go [hunk 1]
```diff
@@ -13,6 +14,21 @@ func (j JobID) String() string {
 	return string(j)
 }
 
+func (j *JobID) UnmarshalJSON(b []byte) error {
+	var asString string
+	err := jsonLib.Unmarshal(b, &asString)
+	if err != nil {
```
Comment (michaljurecko) [inline pkg/keboola/queue_job.go:20]: dalo by sa to detekovat aj takto `len(b) > 0 && b[0] == '"'` -> je to JSON string.

**Review by michaljurecko — APPROVED** [review]
LGTM

**Pull request merged**